library/spdm_transport_tcp_lib/libspdm_tcp_tcp.c

```diff
@@ -0,0 +1,219 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2025 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#include "library/spdm_transport_tcp_lib.h"
+#include "internal/libspdm_common_lib.h"
+#include "hal/library/memlib.h"
+#include "industry_standard/tcp.h"
+
+/**
+ * Get sequence number in an SPDM secure message.
+ *
+ * This value is transport layer specific.
+ *
+ * @param sequence_number        The current sequence number used to encode or decode message.
+ * @param sequence_number_buffer  A buffer to hold the sequence number output used in the secured message.
+ *                             The size in byte of the output buffer shall be 8.
+ *
+ * @return size in byte of the sequence_number_buffer.
+ *        It shall be no greater than 8.
+ *        0 means no sequence number is required.
+ **/
+uint8_t libspdm_tcp_get_sequence_number(uint64_t sequence_number,
+                                        uint8_t *sequence_number_buffer)
+{
+    return 0;
+}
+
+/**
+ * Return max random number count in an SPDM secure message.
+ *
+ * This value is transport layer specific.
+ *
+ * @return Max random number count in an SPDM secured message.
+ *        0 means no randum number is required.
+ **/
+uint32_t libspdm_tcp_get_max_random_number_count(void)
+{
+    return 0;
+}
+
+/**
+ * This function translates the negotiated secured_message_version to a DSP0277 version.
+ *
+ * @param  secured_message_version  The version specified in binding specification and
+ *                                  negotiated in KEY_EXCHANGE/KEY_EXCHANGE_RSP.
+ *
+ * @return The DSP0277 version specified in binding specification,
+ *         which is bound to secured_message_version.
+ */
+spdm_version_number_t libspdm_tcp_get_secured_spdm_version(
+    spdm_version_number_t secured_message_version)
+{
+    return secured_message_version;
```

Comment:
The specification does not define this. I filed https://github.com/DMTF/SPDM-WG/issues/4095.

Reply:
I have returned `SECURED_SPDM_VERSION_12` would that be the right way ?